### PR TITLE
fix: Updated hackathon readme landing page path

### PR DESCRIPTION
**Pull request by jeremytchang**
Updated to match what is running in production

### CODE REVIEW

File: packages/hackathon/README.md [hunk 1]
```diff
@@ -60,7 +81,7 @@ application: hackathon_app {
 Remember to add a model to the project that has a valid connection.
 
 ## Specific steps to `yarn`
-1. run `yarn install` in sdk-codegen 
+1. run `yarn install` in sdk-codegen
 2. run `yarn build` in sdk-codgen
```
Comment (jkaster) [inline packages/hackathon/README.md:85]:
```suggestion
2. run `yarn build` in sdk-codegen
```

File: packages/hackathon/src/data/add_user/sagas.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ const HACKATHON_USER_ATTR_NAME = 'hackathon' // case sensitive
 const HACKATHON_USER_ATTR_LABEL = 'Hackathon'
 const HACKATHON_GROUP_PREFIX = 'Looker_Hack: '
 const LANDING_PAGE_ATTR_NAME = 'landing_page'
-const LANDING_PAGE_PATH = '/extensions/hackathon::hackathon_app'
+const LANDING_PAGE_PATH = '/extensions/hackathon::hackathon'
```
Comment (jkaster) [inline packages/hackathon/src/data/add_user/sagas.ts:53]: Is the user registration script setting this path for the default landing page? Are all existing registrants going to have their homepage updated to this new URL?

**Review by jkaster — COMMENTED** [review]
A suggestion and a  question

File: packages/hackathon/src/data/add_user/sagas.ts (hunk 2 above)
Comment (jeremytchang) [reply packages/hackathon/src/data/add_user/sagas.ts:53]: The user registration group creates a group for hackathon 2021. The script assigns a role  to the group that defines the landing page. When users are added, they are added to the group and thus get the landing page attribute overwritten by the group role. 
So technically, this change shouldn't be needed. And i can jsut change our manifest.kml to rename to `hackathon_app`. But i want to be safe.

**Review by jkaster — DISMISSED** [review]
LGTM

**Review by jkaster — APPROVED** [review]
LGTM

**Pull request merged**